front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -83,14 +83,7 @@ export default function ManageTrainSchedule() {
     ),
     withWarning: rollingStockID === undefined,
     label: t('tabs.rollingStock'),
-    content: rollingStock ? (
-      <RollingStockSelector
-        rollingStockSelected={rollingStock}
-        image={<RollingStock2Img rollingStock={rollingStock} />}
```

Comment:
I have removed the `rollingStockSelected` and `image` props as they are overridden within the component itself. This ensures clarity and avoids redundancy in prop assignments."